src/renderer/models/types/networkMetrics.ts

```diff
@@ -0,0 +1,18 @@
+import {ContainerType, NumberUintType, CompositeArrayType} from "@chainsafe/ssz";
+import {StringType} from "./basic";
+import {NetworkMetric, NetworkMetrics} from "../networkMetrics";
+
+export const NetworkMetricType = new ContainerType<NetworkMetric>({
+    fields: {
+        url: new StringType(),
+        code: new NumberUintType({byteLength: 32}),
+        latency: new NumberUintType({byteLength: 32}),
+        time: new NumberUintType({byteLength: 32}),
+    },
+});
+
+export const NetworkMetricsType = new ContainerType<NetworkMetrics>({
+    fields: {
+        records: new CompositeArrayType({elementType: NetworkMetricType}),
```

Comment:
Don't think this works.
CompositeArrayType doesn't have deserialize/serialize correctly implemented.

Reply:
I can return back to List and open an issue to optimize this?

Reply:
the issue at #348